src/cdk/tree/navigator.ts

```diff
@@ -0,0 +1,311 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {FocusableOption} from '@angular/cdk/a11y';
+import {Directionality} from '@angular/cdk/bidi';
+import {
+  DOWN_ARROW,
+  END,
+  ENTER,
+  HOME,
+  LEFT_ARROW,
+  NUMPAD_MULTIPLY,
+  RIGHT_ARROW,
+  TAB,
+  UP_ARROW
+} from '@angular/cdk/keycodes';
+import {Directive, Optional} from '@angular/core';
+import {Subject} from 'rxjs';
+import {TreeControl} from './control/tree-control';
+
+
+/**
+ * Keyboard navigator interface.
```

Comment:
Expand the class description to elaborate more on what this class does?

We should also probably make it docs-private for now